common/python/configs.py

```diff
@@ -241,6 +264,10 @@ def __init__(self, name, number, type, description, extra_config):
         #: Store the extension register info
         self.extension = extra_config.pop("extension", None)
         self.extension_reg = extra_config.pop("extension_reg", None)
+        self.extension_write = extra_config.pop("extension_write", "")
+        self.extension_read = extra_config.pop("extension_read", "")
+        self.extension_nums = []
+        self.no_config = 0
```

Comment:
is there a reason why this couldn't be `bool` instead? I think it would be easier to read